Neos.Fusion/Classes/View/FusionView.php

```diff
@@ -139,13 +175,16 @@ public function setFusionPathPatterns(array $pathPatterns)
     /**
      * Render the view
      *
-     * @return mixed The rendered view
+     * @return ResponseInterface The rendered view
      * @api
      */
-    public function render()
+    public function render(): ResponseInterface
```

Comment:
We discussed that we want to return PsrStream | PsrResponse here, which will be enforced by https://github.com/neos/flow-development-collection/pull/3286

that way its less breaking for simple string renderings as the PsrStream is stringable.